integ-tests/src/main/java/com/google/base/TestBase.java

```diff
@@ -51,16 +51,26 @@
    * @return Date as a timestamp as Long.
    */
   public static long getTimestampByDate(String dateString) {
+    final String dateFormat = "yyyy-MM-dd HH:mm:ss.SSS";
```

Comment:
I'd either move that to a class level constant or rather use it right in place since it is only used once, but with a parameter comment: `return getTimestampByCustomDate(dateString, /*dateFormat=*/"yyyy...");`

Reply:
Answered below.